### PR TITLE
Add release GH Action

### PR DESCRIPTION
**Pull request by not4Pedro**
This workflow created this release https://github.com/not4Pedro/taranis-test-release/releases/tag/0.0.3
It prepends the multiline tag_message to a full change log.
If no tag message is provided, it falls back to the last commit message (by design).
The release name is implicitly taken from the pushed tag.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        uses: a7ul/tar-action@v1.2.0
+        id: compress
+        with:
+          command: c
+          files: .
+          outPath: ${{ runner.temp }}/${{ env.tag }}.tar.gz
+
+      - name: Create zip of repository
+        run: zip -r ${{ runner.temp }}/${{ env.tag }}.zip .
+
```
Comment (b3n4kh) [inline .github/workflows/release.yaml:39]: Tags always include a tar of the code so this is obsolete, and should be removed

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+name: Release on pushed tags
+
+on:
+  push:
+    tags:
+      - '*'
```
Comment (b3n4kh) [inline .github/workflows/release.yaml:6]: `[0-9]+.[0-9]+.[0-9]+`

**Review by b3n4kh — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      #   if: startsWith(github.ref, 'refs/tags/')
+      #   with:
+      #     tag_name: ${{ env.tag }}
+      #     release_name: ${{ env.tag }}
+      #     body: ${{ github.event.head_commit.message }}
+      #     files: |
+      #       docker/compose.yml
+      #       ${{ runner.temp }}/env.sample
+      #       ${{ runner.temp }}/${{ env.tag }}.zip
+      #       ${{ runner.temp }}/${{ env.tag }}.tar.gz
```
Comment (b3n4kh) [inline .github/workflows/release.yaml:45]: remove these two lines

**Comment by not4Pedro** [comment]
This workflow created this release https://github.com/not4Pedro/taranis-test-release/releases/tag/0.0.3 
It prepends the multiline `tag_message` to a full change log. 
If no tag message is provided, it falls back to the last commit message (by design).
The release name is implicitly taken from the pushed tag.

**Pull request merged**